test/load/src/main.rs

```diff
@@ -37,6 +41,80 @@ async fn signup_login(user: &mut GooseUser) -> TransactionResult {
     Ok(())
 }
 
+async fn get_listing(user: &mut GooseUser) -> TransactionResult {
+    // get random listing id between 1-2000
```

Comment:
```suggestion
    // get random listing id between 1-2000
```
Lets get rid of these copilot comments

Reply:
These aren't copilot lol I just added them when I was blocking out the function, but sure